.env.example

```diff
@@ -72,9 +72,9 @@ NEXT_PUBLIC_ROUND_ID="open-rpgf-1"
 NEXT_PUBLIC_TOKEN_NAME="OP"
 
 # How many votes each ballot can hold
-NEXT_PUBLIC_MAX_VOTES_TOTAL=30000000
+NEXT_PUBLIC_MAX_VOTES_TOTAL=150
 # How many votes each project can receive
-NEXT_PUBLIC_MAX_VOTES_PROJECT=5000000
+NEXT_PUBLIC_MAX_VOTES_PROJECT=50
```

Comment:
what is this? like the max vote it can receive?

Reply:
Right, it's max vote weight per project vote.

Reply:
shouldn't we be taking it from the initial voice credit balance of the user which they received upon signing up? Also, one could put all their weight to 1 project, so the total and per project should be the same imo

Reply:
I think yes, it's possible to do it.

Reply:
Yes, let's merge this one because I need to add some changes for maci-cli to get the balance.